packages/appeals-service-api/src/services/back-office-v2/formatters/index.js

```diff
@@ -0,0 +1,33 @@
+const { APPEAL_ID } = require('@pins/business-rules/src/constants');
+/**
+ * @typedef {import('src/spec/api-types').AppealSubmission} AppealSubmission
+ * @typedef {import('../../../routes/v2/appeal-cases/_caseReference/lpa-questionnaire-submission/questionnaire-submission').LPAQuestionnaireSubmission} LPAQuestionnaireSubmission
+ */
+
+const appealTypeToDirName = {
+	[APPEAL_ID.HOUSEHOLDER]: 'has'
+};
+
+/**
+ * this does "work" (the type is visible in autocomplete etc)
+ * we should probably just tighten up what we expect to be in the db
+ * @type {{
+ *   appeal: Record<AppealSubmission['appeal']['appealType'], (appeal: AppealSubmission['appeal']) => *>,
+ *   questionnaire: Record<AppealSubmission['appeal']['appealType'], (caseReference: string, questionnaireResponse: LPAQuestionnaireSubmission) => *>
+ * }}
+ */
+module.exports = ['appeal', 'questionnaire'].reduce(
```

Comment:
this reduces the adjacent directories to an object the `BackOfficeThingy` class can navigate like `formatter['questionnaire']['1001']` to get a function to format the relevant data. That's gunna mean that as we build out more types we don't need conditions in the `BackOfficeThingy` because the data will tell us that number and the method being used tells us what kind thing we need to format.